crates/bitwarden-crypto/Cargo.toml

```diff
@@ -46,8 +46,13 @@ uniffi = { version = "=0.28.1", optional = true }
 uuid = { version = ">=1.3.3, <2.0", features = ["serde"] }
 zeroize = { version = ">=1.7.0, <2.0", features = ["derive", "aarch64"] }
 
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+memsec = { version = "0.7.0", features = [
+    "alloc_ext",
+], git = "https://github.com/dani-garcia/memsec", rev = "3d2e272d284442637bac0a7d94f76883960db7e2" }
```

Comment:
`memsec` is using `windows-sys` 0.45, which was causing build errors similar to the ones fixed by #1053.

Using the latest `windows-sys` 0.59 seems to solve the problem for me. TODO: Open a PR upstream?